README.md

```diff
@@ -9,6 +9,33 @@
 
 Snowplow Fractribution (marketing attribution) model for dbt
 
+### Setup steps
+
+1. Configure the `conversion_clause` macro to filter your raw Snowplow events to successful conversion events.
+2. Configure the `conversion_value` macro to return the value of the conversion event.
+3. Configure the default `channel_classification` macro to yield your expected channels. The ROAS calculations / attribution calculations will run against these channel definitions.
+
+### Running
+
+1. Ensure the setup steps have been completed above.
+2. Run `dbt run`, or `dbt run --select package:fractribution`
+
+
```

Comment:
Can we have an Adapter support section as well here, please? It is important for users to know it is for Snowflake and for which version etc. See example, [here](https://github.com/snowplow/dbt-snowplow-web/blob/main/README.md).

Reply:
Added, and also updated the Readme to make it more like the snowplow-web one. Let me know if I should change anything